src/api/comments.js

```diff
@@ -0,0 +1,18 @@
+import { add, dell, get } from './api';
+
+const getComments = () => get('/comments');
+
+export const getPostComments = postId => getComments()
+  .then((result) => {
+    const comments = result.filter(comment => comment.postId === postId);
+
+    if (comments === undefined) {
+      return [];
+    }
+
+    return comments;
+  });
+
```

Comment:
You don't need to load all comments (a;most 5000 of them) everytime.
Check the PARAMS section of API documentation.
https://mate-academy.github.io/fe-students-api/#params